vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -21,22 +21,23 @@ If you're already comfortable setting up an EKS cluster and installing NGINX, th
 - AWS account
 - kubectl
 
-## Create and Connect the EKS Cluster 
+## Create and Connect to the EKS Cluster 
 These steps use the AWS console to create the cluster. Experienced AWS users may choose to use the equivalent CLI commands.
 
 ### Configure an Identity and Access Management (IAM) Role
 1. [Login to AWS](http://console.aws.amazon.com/) as a non-`super admin` user
    - It is not recommended to use a `super admin` account since clusters created that way may have restrictions that complicate your installation.
-   - The user account needs access to EKS and the minimum permissions to create a cluster. You may need additional policies for Amazon Route 53 or other services, depending on your configuration.
-2. Create an IAM role for the cluster so that AWS can provision assets 
+   - The user account needs access to EKS and the minimum permissions to create a cluster including the ability to view/update Addons for the cluster. You may need additional policies for Amazon Route 53 or other services, depending on your configuration.
```

Comment:
"...cluster, including..." (comma for pause / improved readability)

add-ons (per gartner's usage)